src/main/java/ladder/domain/LadderLine.java

```diff
@@ -5,29 +5,66 @@
 import java.util.Iterator;
 import java.util.List;
 import java.util.Objects;
+import java.util.stream.Collectors;
 import java.util.stream.IntStream;
+import java.util.stream.Stream;
 
 public class LadderLine implements Iterable<Rope> {
+    private static final int FIRST_ROPE_INDEX = 0;
     private final List<Rope> ropes;
```

Comment:
```suggestion
    private static final int FIRST_ROPE_INDEX = 0;
    
    private final List<Rope> ropes;
```

클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다 :)